mesonbuild/backend/vs2010backend.py

```diff
@@ -118,67 +118,73 @@ def __init__(self, build: T.Optional[build.Build], interpreter: T.Optional[Inter
     def get_target_private_dir(self, target):
         return os.path.join(self.get_target_dir(target), target.get_id())
 
+    def generate_genlist_for_target(self, genlist, target: build.BuildTarget, parent_node, generator_output_files, custom_target_include_dirs, custom_target_output_files) -> None:
```

Comment:
Even if this file is not run through mypy (yet), I think you should anotate all the function arguments, to help the reviewer.